day34/validateIP.py

```diff
@@ -0,0 +1,74 @@
+def validateIP(ip):
+    ipv4 = ip.split(".")
+    ipv6 = ip.split(":")
+    if len(ipv4) == 4 and validateIPv4(ipv4):
```

Comment:
imo i don't like your validate functions taking lists of segments. it feels a bit unnatural to me, why not just send in the direct string?